pkg/lib/cockpit-po-plugin.js

```diff
@@ -105,7 +103,17 @@ function init(options) {
 }
 
 function run(webpack_module, webpack_compilation) {
-    return Promise.all(get_po_files().map(f => buildFile(f, webpack_module, webpack_compilation)));
+    return Promise.all(get_po_files().map(f => {
+        if (webpack_compilation)
+            webpack_compilation.fileDependencies.add(f);
+        const lang = path.basename(f).slice(0, -3);
+        return Promise.all([
+            buildFile(f, webpack_module, webpack_compilation,
+                      `${config.subdir}po.${lang}.js`, str => !str.startsWith(`pkg/${config.subdir}manifest.json`)),
+            buildFile(f, webpack_module, webpack_compilation,
+                      `${config.subdir}po.manifest.${lang}.js`, str => str.startsWith(`pkg/${config.subdir}manifest.json`))
```

Comment:
I initially did not understand why this is `.startsWith()` instead of `.includes()`. When I have a translation like
```
#: src/components/vm/vmDetailsPage.jsx:112    
#: src/components/vm/vmDetailsPage.jsx:265 src/manifest.json:0    
msgid "Virtual machines"    
msgstr "Virtuelle Maschinen"  
```
then I want *both* LL.po.js and LL.manifest.po.js to contain this string. However, buildFile() splits references into words.

But this is still wrong -- they are in pkg/subdir only for cockpit, but not for starter-kit and friends. Let's rather not make assumptions about the directory. How about `.endsWith('.manifest.json`)` instead? That entirely avoids making assumptions about the location.